src/README.md

```diff
@@ -567,3 +567,135 @@ Source:
+| --------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| (Shared among all routines) | `0x0000001` | Routine passed.                                                                                                                                                     |
+| (Shared among all routines) | `0x0000002` | (Shared status code for exceptions; refer to each routine for exact messages)                                                                                        |
+| AC Power                    | `0x0010001` | Expected online power supply, found offline power supply.                                                                                                           |
+| AC Power                    | `0x0010002` | Expected offline power supply, found online power supply.                                                                                                           |
+| AC Power                    | `0x0010003` | Read power type different from expected power type.                                                                                                                 |
+| AC Power                    | `0x0000002` | No valid AC power supply found.                                                                                                                                     |
+| Battery Capacity            | `0x0070001` | Invalid BatteryCapacityRoutineParameters.                                                                                                                           |
+| Battery Capacity            | `0x0070002` | Battery design capacity within given limits.                                                                                                                        |
+| Battery Capacity            | `0x0070003` | Battery design capacity not within given limits.                                                                                                                    |
+| Battery Capacity            | `0x0000002` | Failed to get power supply properties from powerd.                                                                                                                  |
+| Battery Charge              | `0x0080001` | Battery is not charging.                                                                                                                                            |
+| Battery Charge              | `0x0080002` | Battery charge percent less than minimum required charge percent.                                                                                                   |
+| Battery Charge              | `0x0000002` | Failed to read battery attributes from sysfs.                                                                                                                       |
```

Comment:
nit: IMO, It would be clearer to put all exception in one row for each routine (at least put together) since It's hard to distinguish between`0x0000002` and `0x00?0002`.

Reply:
Good idea. I reorg the table, and group and bold the exceptions in routines.